ledger/catchpointwriter.go

```diff
@@ -79,14 +85,18 @@ type encodedBalanceRecordV6 struct {
 	Address     basics.Address      `codec:"a,allocbound=crypto.DigestSize"`
 	AccountData msgp.Raw            `codec:"b,allocbound=basics.MaxEncodedAccountDataSize"`
 	Resources   map[uint64]msgp.Raw `codec:"c,allocbound=basics.MaxEncodedAccountDataSize"`
+
+	// flag indicating whether there are more records for the same account coming up
+	ExpectingMoreEntries bool `codec:"e"`
```

Comment:
So, if a client is running a version that doesn't have this change, they will get a message about "unrecognized msgp field e" or similar? Seems fine ... especially if there is a consensus upgrade coming up